libs/designer/src/lib/core/DesignerProvider.tsx

```diff
@@ -54,7 +55,7 @@ export const DesignerProvider = ({ key, locale = 'en', options, children }: Desi
               <div data-color-scheme={themeName} className={`msla-theme-${themeName}`} style={{ height: '100vh', overflow: 'hidden' }}>
                 <ReactQueryProvider>
                   <IntlProvider locale={locale} defaultLocale={locale} onError={onError}>
-                    <ReduxReset key={key} />
+                    <ReduxReset id={id} />
```

Comment:
I'm not actually sure why this change is showing up on this PR, I made this change last week. It does reset redux the same way, the problem is that key shouldn't be used as a prop, it can have problems because of how it works with react already.

Reply:
yep it does at least in ladesigner, it seems localdesigner discard was never implemented? all it does is reset the dirty state. also do you know what partner team uses this?